packit_service/worker/handler.py

```diff
@@ -63,6 +63,12 @@ def report(
             self.commit_sha, state, url, description, "packit/rpm-build"
         )
 
+    def reset_status(self, state: str, description: str):
```

Comment:
nit: shouldn't this be set_status?

Reply:
reset mean, get rid of it to initial state. Will change it.

Reply:
ah, okay, didn't know it would reset the state since the state is an argument